pkg/provider/provider.go

```diff
@@ -217,12 +219,32 @@ var allowAllOrigins = func(_ string) bool {
 }
 
 // AuthCallbackURL builds the url for the redirect (with the requestID) after a successful login
-func AuthCallbackURL(p *Provider) func(context.Context, string) string {
+func (p *Provider) AuthCallbackURL() func(context.Context, string) string {
```

Comment:
just noticed now: any particular reason to switch from function to method?
(breaking change)

Reply:
To make it consistent to the OIDC usage, as I saw the difference

Reply:
but isn't it a function as well?
https://github.com/zitadel/oidc/blob/main/pkg/op/op.go#L150-L154

Reply:
This function is what I mean, do we have something else to change?
https://github.com/zitadel/oidc/blob/cf6ce69d79666cef2ac429607286c3aa63bcb677/pkg/op/server_legacy.go#L79

Reply:
https://github.com/zitadel/zitadel/blob/83bdaf43c35b7c4c5dfaf3e822a917e95d9f5fae/cmd/start/start.go#L532